.github/workflows/benchmark.yml

```diff
@@ -47,10 +47,12 @@ jobs:
         uses: ./.github/actions/setup
         with:
           rust-version: ${{ steps.rust-version.outputs.rust_version}}
-      - name: Compile benchmarks
-        run: cargo bench --no-run
-      - name: Run benchmark
+      - name: Compile benchmarks with all features
+        run: cargo bench --no-run --all-features
+      - name: Run benchmarks with no features
         run: cargo bench --benches -p ${{ matrix.package }} -- --output-format bencher | tee ${{ matrix.package }}.txt
+      - name: Run benchmarks with iouring feature
```

Comment:
I don't think this will work with our existing benchmarks pipeline (I think it'll just show up in the same charts as the non-iouring stuff).

I think we need to update the `matrix` with "features" and update the `name` in the publish step to include said-features.